### PR TITLE
[COST-5637] Change tag exclusions to an or operation for openshift-virt

### PR DESCRIPTION
**Pull request by myersCody**
## Jira Ticket

[COST-5637](https://issues.redhat.com/browse/COST-5637)

## Description

This change will change the operation we use for tag exclusions for the OpenShift virtualization endpoint.

## Testing

1. Check out branch
2. Load test customer data for on prem
3. Enable the `exclude` & `excluded` tag key through the settings
4. Resummarize on prem source.
5. Hit this endpoint and notice the key,value pairs are excluded from the return:
- http://localhost:8000/api/cost-management/v1/reports/openshift/resources/virtual-machines/?limit=100&exclude[or:tag:exclude]=C, D&exclude[or:tag:excluded]=sushi, fedora
6. Go to main and realize nothing is returned.

## Release Notes
- [ ] proposed release note

```markdown
* [COST-5637](https://issues.redhat.com/browse/COST-5637) Tweak multiple tag key value pair exclusion logic
```

### CODE REVIEW

**Review by lcouzens — DISMISSED** [review]
Great job Cody!

File: koku/api/report/test/test_queries.py [hunk 1]
```diff
@@ -586,28 +590,84 @@ def test_apply_group_null_label_empty_string(self):
         out_data = handler._apply_group_null_label(data, groups)
         self.assertEqual(expected, out_data)
 
-    # FIXME: need test for _apply_group_by
```
Comment (myersCody) [inline koku/api/report/test/test_queries.py:589]: These are all already tested by our api tests

Comment (lcouzens) [reply koku/api/report/test/test_queries.py:589]: Thanks for cleaning these up.

**Review by lcouzens — APPROVED** [review]
Nice job!

**Pull request merged**